docker/tests/factories/mock_serper_response.py

```
@@ -0,0 +1,37 @@
+        self._json_data = {"organic": organic_results or []}
+
+    def json(self) -> Dict[str, Any]:
+        """Return JSON response data."""
+        return self._json_data
+
+    def raise_for_status(self):
+        """Raise HTTPError if status code indicates error."""
+        if self.status_code >= 400:
+            import requests
+
+            raise requests.HTTPError(f"{self.status_code} Error")
```

Ruff (0.13.2)

Avoid specifying long messages outside the exception class

(TRY003)